hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -131,6 +131,32 @@ <h2 class="formH2">Your application has been submitted!</h2>
                 {% endif %}
             </div>
 
+            {% if application is not none %}
+            <div class="borderTopDiv z-depth-3">
+                <h2 class="formH2" id="faq">Application Answers</h2>
+
+                <p class="applicationQuestionAnswer">Birthday: {{application.birthday}}</p>
+
+                <p class="applicationQuestionAnswer">Ethnicity: {{application.ethnicity}}</p>
+
+                <p class="applicationQuestionAnswer">Phone Number: {{application.phone_number}}</p>
+
+                <p class="applicationQuestionAnswer">School: {{application.school}}</p>
+
+                <p class="applicationQuestionAnswer">Study Level: {{application.study_level}}</p>
+
+                <p class="applicationQuestionAnswer">Graduation Year: {{application.graduation_year}}</p>
+
+                <p class="applicationQuestionAnswer">First question: {{application.q1}}</p>
+                
+                <p class="applicationQuestionAnswer">Second question: {{application.q2}}</p>
+
+                <p class="applicationQuestionAnswer">Third question: {{application.q3}}</p>
+                
+                <p class="applicationQuestionAnswer">Resume: {{application.resume}}</p>
```

Comment:
@YuYing-Liang  @Mustaballer  Do you guys think it is necessary to allow ppl to view their submitted resumes? If not, we can just show the text here as how Terry currently implemented here. If we allow, @terryluan12 you might need to change it to the link that can direct to their resume file.

Reply:
We should link the resume on the page and the user can open it in a new tab.